### PR TITLE
Remove setarch command in vfc_ddebug 

### PR DESCRIPTION
**Pull request by yohanchatelain**
- Uses [personality](http://man7.org/linux/man-pages/man2/personality.2.html) function with ADDR_NO_RANDOMIZE
> ADDR_NO_RANDOMIZE (since Linux 2.6.12)
              With this flag set, disable address-space-layout
              randomization.
- Allows to run vfc_ddebug without sudo mode

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ install:
   - export PYTHONPATH=/usr/local/lib/python${PYTHON_VERSION}/site-packages/:$PYTHONPATH
 
 script:
-  - sudo -E env PYTHONPATH=$PYTHONPATH LD_LIBRARY_PATH=$LD_LIBRARY_PATH make installcheck
+  - make installcheck
```
Comment (pablooliveira) [inline .travis.yml]: nice !

File: src/vfc_ddebug [hunk 2]
```diff
@@ -1,9 +1,7 @@
-#!/bin/sh
-''''exec setarch $(uname -m) -R python3 -u "$0" "$@" #'''
-# This hack is an ugly but portable alternative to #!/usr/bin/env -S python3 -u
-# we call setarch -R to disable ASLR, so that addresses during reference run
```
Comment (pablooliveira) [inline src/vfc_ddebug]: can you move this comment to the function disable_ASLR to explain why this is needed during delta_debug

File: Dockerfile [hunk 3]
```diff
@@ -53,57 +53,7 @@ RUN git clone --depth=1 https://github.com/verificarlo/verificarlo.git &&  \
...
-RUN git clone --depth=1 https://github.com/verificarlo/verificarlo.git &&  \
-    cd verificarlo && \
-    ./autogen.sh && \
-    ./configure --with-llvm=${LLVM_INSTALL_PATH} --with-dragonegg=${DRAGONEGG_PATH} CC=gcc-${GCC_VERSION} CXX=g++-${GCC_VERSION} || cat config.log 
-
-# Build and test verificarlo
-RUN cat $HOME/.bashrc && \
-    cd verificarlo && \
-    make && sudo make install && \
-    sudo -E env PYTHONPATH=$PYTHONPATH LD_LIBRARY_PATH=$LD_LIBRARY_PATH make installcheck
```
Comment (pablooliveira) [inline Dockerfile]: How come you can remove all this ?

Comment (yohanchatelain) [reply Dockerfile]: It was a mistake. In the previous commit, I duplicated this code so I removed it in my second commit so github compares this version with the first commit but actually line 56 has changed.

File: src/vfc_ddebug (hunk 2 above)
Comment (yohanchatelain) [reply src/vfc_ddebug]: Done.

Comment (yohanchatelain) [reply src/vfc_ddebug]: Done.

File: src/vfc_ddebug [hunk 4]
```diff
@@ -28,6 +28,9 @@ class DDline(DD_stoch.DDStoch):
     def coerce(self, delta_config):
         return "\n".join([l[:-1] for l in delta_config])
 
+# This hack is an ugly but portable alternative to #!/usr/bin/env -S python3 -u
```
Comment (pablooliveira) [inline src/vfc_ddebug]: Please remove line 31.

**Comment by pablooliveira** [comment]
Thanks :tada:

**Pull request merged**